universal/ubi8/Dockerfile

```diff
@@ -378,6 +378,32 @@ cd -
 rm -rf "${TEMP_DIR}"
 EOF
 
+# e2fsprogs setup
+# Since e2fsprogs-static package has removed RHEL 8 distribution, it is not possible to install from the repository
+# https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/considerations_in_adopting_rhel_8/index#removed-packages_changes-to-packages
+RUN <<EOF
+set -euf -o pipefail
+TEMP_DIR="$(mktemp -d)"
+cd "${TEMP_DIR}"
+E2FSPROGS_VERSION="1.46.5"
+E2FSPROGS_TGZ="e2fsprogs-${E2FSPROGS_VERSION}.tar.gz"
```

Comment:
if you need this in downstream UDI image, you'll need a 'prefetch and reuse' approach. See https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-udi/get-sources.sh#L76-L92

Reply:
yep, in downstream UDI this tarball will be prefetched for offline build